### DISCUSSION TITLE
Welcome to coin Discussions!

### DISCUSSION THREAD

**Discussion by VolkerEnderlein** [coin3d/coin#478] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yves78** [comment]
Thanks a lot !
 I'm not a very skilled programmer, and I tried since a long time to build Coin, but got a lot of problems with the build chain and the versions.
Then I just found your binaries, and I was really very happy !
Regards.

**Comment by ayerbeg** [comment]
Installation instructions needs an update to indicate the cmake option to install cpack.d directory.

**Comment by gxxwfyhjs** [comment]
Btw w for the creators
